tmt/base.py

```diff
@@ -542,16 +542,18 @@ def export(self, format_='yaml', keys=None):
             tmt.export.export_to_nitrate(self)
 
         # Common node export otherwise
-        elif keys == 'fmf-id':
-            if format_ == 'dict':
+        elif format_ == 'dict' or format_ == "yaml":
+            if keys == 'fmf-id':
                 return self.fmf_id
-            elif format_ == "yaml":
-                return tmt.utils.dict_to_yaml(self.fmf_id, start=True)
             else:
-                raise tmt.utils.GeneralError(
-                    f"Invalid test export format '{format_}'.")
+                data = {}
+                data['name'] = self.name
```

Comment:
Instead of adding the `name` key here, let's update the `Core.export()` method instead to always include the `name` key unless an explicit list of the desired `keys` is provided as the parameter. We can then remove similar action from the `Plan.export()` method as well and `Test.export()` will need almost no change.

Reply:
Thanks.